types/src/state_store/state_key.rs

```diff
@@ -154,6 +154,20 @@ impl StateKey {
     pub fn get_shard_id(&self) -> u8 {
         CryptoHash::hash(self).nibble(0)
     }
+
+    pub fn is_aptos_path(&self) -> bool {
+        use move_core_types::{account_address::AccountAddress, language_storage::CODE_TAG};
+        match self.inner() {
+            StateKeyInner::AccessPath(access_path) => {
+                !access_path.path.is_empty()
+                    && access_path.path[0] == CODE_TAG
```

Comment:
I think you can check if access path is code using `is_code` function?

Reply:
done